addons/web/static/src/core/py_js/py_parser.js

```diff
@@ -118,6 +120,109 @@ function isSymbol(token, value) {
     return token.type === 2 /* Symbol */ && token.value === value;
 }
 
+/**
+ * Check if a token is in the current tokens scope (scoped by symbol {} or [] or ())
+ *
+ * @param {Token} token
```

Comment:
incomplete (and unclear) docstring

Reply:
should be ok now (I also removed useless extra parameters)